benches/storage/engine/bench_btree_engine.rs

```diff
@@ -0,0 +1,94 @@
+// Copyright 2018 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use super::{engine_get_bench, engine_put_bench, engine_snapshot_bench, ITERATIONS};
+use test::Bencher;
+use tikv::storage::engine::BTreeEngine;
+
+#[bench]
+fn bench_engine_get_btree_1000_times_from_000_000_size(bencher: &mut Bencher) {
```

Comment:
Prefer to use Criterion when multiple sizes are involved. See https://github.com/tikv/tikv/pull/3477 for an example.